enter.c

```diff
@@ -263,8 +287,6 @@ int enter(struct entry_settings *opts)
 		apply_climits(subcgroupfd, opts->climits);
 
 		close(subcgroupfd);
-	} else if (opts->nactiveclimits != 0 && opts->cgroup_path == NULL) {
-		errx(1, "unable to apply limits without --cgroup specified");
```

Comment:
This is likely still useful when we cannot determine the root cgroup for whatever reason

Reply:
Added this back in case parsing /proc/self/cgroup does not turn anything up.